lib/modules/userTagger.js

```diff
@@ -740,15 +740,25 @@ let currentSortMethod, isDescending;
 async function drawUserTagTable(sortMethod, descending) {
 	currentSortMethod = sortMethod || currentSortMethod;
 	isDescending = (descending === undefined || descending === null) ? isDescending : descending;
-	const filterType = $('#tagFilter').val();
 	const tagsData = await tagStorage.getAll();
-	const tags = filterMap(Object.entries(tagsData), ([k, v]) => {
+	const filterType = $('#tagFilter').val();
+	let tags = filterMap(Object.entries(tagsData), ([k, v]) => {
 		const tag = Tag.getUnfilled(k);
 		tag.load(v);
 		if (filterType === 'tagged users' && !tag.text) return;
```

Comment:
I didn't notice this earlier, but `filterType === 'tagged users'` won't work when the string is translated as the options in the select does not have the `value` attribute defined.